src/openrct2/actions/RideSetPriceAction.hpp

```diff
@@ -181,7 +181,7 @@ DEFINE_GAME_ACTION(RideSetPriceAction, GAME_COMMAND_SET_RIDE_PRICE, GameActions:
             {
                 // If the shop item is the same or an on-ride photo
                 if (rideEntry->shop_item[1] == shopItem
-                    || (rideEntry->shop_item[1] == SHOP_ITEM_NONE && ShopItems[shopItem].IsPhoto()))
+                    || (rideEntry->shop_item[1] == ShopItem::None && ShopItems[EnumValue(shopItem)].IsPhoto()))
```

Comment:
`ShopItems[EnumValue(` will be used in a lot of locations. Make an accessor function.

```
GetShopItemDescriptor(ShopItem item)
{
    return ShopItems[EnumValue(item)];
}```